ang/crmMailing/BlockMailing.html

```diff
@@ -79,5 +79,19 @@
         placeholder="Subject"
         name="subject" />
     </div>
+    <div ng-if="crmMailingConst.isMultiLingual">
+      <div crm-ui-field="{name: 'subform.language', title: ts('Language')}">
+        <select
+          crm-ui-id="subform.language"
+          crm-ui-select="{dropdownAutoWidth : true, allowClear: false}"
+          name="language"
+          ng-model="mailing.language"
+          required
+          >
+          <option value=""></option>
```

Comment:
There will be, at the very least, an unsubscription/opt-out token. If that token leads to an URL, then the language needs to be set. In my opinion the language should be mandatory. Alternatively, we could default to the current UI language?